grails-app/views/index.gsp

```diff
@@ -3,22 +3,22 @@
 <head>
     <meta name="layout" content="${grailsApplication.config.skin.layout}"/>
     <meta name="breadcrumb" content="Search"/>
-    <title>${grailsApplication.config.skin?.orgNameLong} - Search</title>
+    <title><g:message code="page.index.title" args="${[ grailsApplication.config.skin.orgNameLong ]}"/></title>
```

Comment:
Just FYI - if I touch any code with `grailsApplication.config.foo.bar.hello` I try and convert to this form, which is much safer (and preferred by the docs as won't do odd things when a key or value is not set): `grailsApplication.config.getProperty('foo.bar.hello')`. (for next time, don't worry about it here).